### PR TITLE
Prepare ServiceBus 7.5.1 December-2021 release

### PR DESCRIPTION
**Pull request by ki1729**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [X] **The pull request does not introduce [breaking changes]**
- [X] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 7.6.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 7.5.1 (2021-12-08)
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md:3]: I think the azure-core version was also updated which should be included in the changelog

File: sdk/servicebus/azure-messaging-servicebus/pom.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-servicebus</artifactId>
-  <version>7.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;current} -->
+  <version>7.5.1</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;current} -->
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/pom.xml:17]: README.md should be updated as well.

File: sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md (hunk 1 above)
Comment (ki1729) [reply sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md:3]: Done

File: sdk/servicebus/azure-messaging-servicebus/pom.xml (hunk 2 above)
Comment (ki1729) [reply sdk/servicebus/azure-messaging-servicebus/pom.xml:17]: Done

**Review by srnagar — APPROVED** [review]

**Review by anuchandy — APPROVED** [review]

**Pull request merged**